lib/doorkeeper/models/concerns/revocable.rb

```diff
@@ -8,6 +8,29 @@ def revoke(clock = Time)
       def revoked?
         !!(revoked_at && revoked_at <= Time.now)
       end
+
+      def revoke_in(time)
+        update_attribute :revoked_at, Time.now + time
+      end
+
+      def revoke_previous_refresh_token!
+        old_refresh_token = AccessToken.by_refresh_token(previous_refresh_token)
+        if old_refresh_token && !old_refresh_token.revoked?
+          if configuration.refresh_token_revoked_in.zero?
+            old_refresh_token.revoke
+          else
+            old_refresh_token.revoke_in(configuration.refresh_token_revoked_in)
+          end
+        end
+
+        if previous_refresh_token.present?
+          update_attribute(:previous_refresh_token, nil)
+        end
+      end
+
+      def configuration
+        Doorkeeper.configuration
+      end
```

Comment:
We define the `configuration` method but don't use it, and we'd use it only once in this class. I'd remove the stray method.